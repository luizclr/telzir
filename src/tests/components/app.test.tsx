import React from "react";
import App from "../../components/app";
import { shallow } from "enzyme";

describe("<App />", () => {
  it("should render component", () => {
    const wrapper = shallow(<App />);

    expect(wrapper).toMatchSnapshot();
  });
});
