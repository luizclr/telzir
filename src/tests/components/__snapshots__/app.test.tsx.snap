// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<App /> should render component 1`] = `
<div
  className="App"
  data-testid="app"
>
  <div
    className="card"
  >
    <Header />
    <Form
      setCallCosts={[Function]}
    />
    <CallCosts
      callCosts={
        Object {
          "valid": false,
          "withPlan": 0,
          "withoutPlan": 0,
        }
      }
    />
  </div>
</div>
`;
